docs/static/breaking-changes.asciidoc

```diff
@@ -70,3 +70,32 @@ Starting with the 2.0 release of Logstash, the default value of the `filter_work
 plugins is half of the available CPU cores, instead of 1. This change increases parallelism in filter execution for
 resource-intensive filtering operations. You can continue to use the `-w` flag to manually set the value for this option,
 as in previous releases.
+
+
+[float]
+=== Ruby filter Changes
+
+With the migration to the Java Event we have changed how we can access the internal data, the object doesn't return a
+reference to the data but a copy instead. This change how you have to do manipulation of your data especially when working with nested hashes.
+When working with nested hashes its recommended to use the `fieldref` syntax instead of using multiple brakets.
```

Comment:
Change to say, "When working with nested hashes, it’s recommended that you use...."